app/src/main/res/layout/snackbar_view.xml

```diff
@@ -0,0 +1,64 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.cardview.widget.CardView xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Cardview shouldn't be the root. Please enclose cardview within another ConstraintLayout. Give padding to this parent layout instead to the cardview.